src/Symfony/Component/Security/Core/Authorization/Voter/AccessTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+trait AccessTrait
+{
+    /** @var int */
+    private $access;
```

Comment:
Mustn't it be protected if it's reused in classes using the trait?

Reply:
No classes implementing this trait can access this private var

Reply:
Maybe add a doc that int is one of the
voterinterface::ACCESS_ Consts